docs/topics/testing/tools.txt

```diff
@@ -1905,6 +1905,12 @@ your test suite.
 
         In older versions, error messages didn't contain the ``haystack``.
 
+.. method:: SimpleTestCase.assertNotInHTML(needle, haystack, msg_prefix='')
```

Comment:
Blank line and `versionadded` annotations are missing. Please add also a release note (`5.1.txt`).

Reply:
You forgot to add the release note.

Reply:
Apologies, I don't know how I could have missed this but I did... I'll try not forget next time, thanks 😅